### ISSUE TITLE
Storage Drawer, External Storage wrong inventory count or missing all together 1.16

### ISSUE THREAD

**Issue by JeffsRealm** [refinedmods/refinedstorage#2880]
#### What you expected to happen:
I expect refined storage to represent accurate number from externeral storage and storage drawers.

#### Steps to reproduce:
1. Connect refined storage to storage drawers. Using external storage bus on a drawer controller. Use the key to show inventory numbers on the drawers to make it easier.
2. Completely shut down the game close everything and start back up again.
3. Look at the counts of items in the refined storage grid compare them to counts on the drawers.
...

#### Version (make sure you are on the latest version before reporting):

- Minecraft: 1:16 I am running the current Let Play Feed The Beast Direwolf 20 1.16 also the latest version of it but I noticed this since I started refined storage in version 1.5.1 of the pack current version of the pack is 1.6.1
- Forge: 
- Refined Storage: Sorry Feed the beast is really difficult to navigate and find versions refinedstorage-1.9.11.jar is best I can do, if I can give you any more information let me know. I know Direwolf is using refined Storage in the series but not doing the whole drawer external storage thing. He is also just getting rolling with.

Does this issue occur on a server? [yes/no] Unknown I am not playing on a server

#### If a (crash)log is relevant for this issue, link it here: Never get a crash.

[pastebin/gist/etc link here]

**Comment by Goldy93** [comment]
Same thing on 1.16.4, forge ver 35.1.37, single player world.  Only happens when relogging into the world.  All I do then is go into the disk drive, switch it to redstone only mode and switch it back and the items in the storage drawers appear again and will remain that way until relogging.

**Comment by bloodscon** [comment]
Is the RS controller & drawer network in the same chunk (F3 + G shows chunk borders). If they are not how many chunks away from the RS controller is the drawer network
and do you have any chunks claimed and load using the FTB utilities

**Comment by JeffsRealm** [comment]
They are in different chunks yes. However, is is only 1 chunk to the south.  They are on opposite corners creating a slightly longer run but not bad I have done a lot worse in the past prior versions. It is also up 14 blocks. Basically I am building a mountain house with more vertical. it is 2x2 chunks and I keep everything contained inside the 4 chunks. I haven't created an animal farm yet but using all indoor growing techniques for plants and things. 

Everything is chunk loaded with the FTB chunk loaders.

**Comment by bloodscon** [comment]
have a look in the pack to see if there is other chunk loader options as this maybe related to the same bug that is in 1.12 with ftb chunk loader and RS 

https://www.reddit.com/r/feedthebeast/comments/dkvuiu/refined_storage_not_detecting_my_storage_drawers/

**Comment by JeffsRealm** [comment]
Yeah I remember that bug, I ran into it as well, I had a feeling you might suggest that, so put a small set up all in the same chunk as the controller. Still have the same issue, the issue only presents itself at logon. I haven't gone on long enough and set up automated inserting and exporting of the small set up for testing so can't tell long term effect. But shutting the game down and starting back up again did have the same time issue I reported above.

As far as additional chunk loaders. I have looked and there is nothing in the pack so I can't tell you that for sure. I also don't know of any good ones that will work with the pack. Used to have Chicken Chunks as a backup in the FTB packs but I don't think it has been worked on yet.  I do have Mekanism Pipes of various types running all over crossing chunks and also a few refined pipes still in place from early game. None have given me an issue. The FTB chunk loader bug from 1.12 I noticed it affecting  EnderIO as well. Ender IO is not available yet and Thermal doesn't have its pipes up and running yet to see if it is affecting them. I do have RFTools builder running which does load its own chunks as it mines. I have had no issue with that.  

I do have a new one to add though, it may help, may not. So twice now since I created this I have logged in to find Refined storage will not take power. It is fully charged but just dies shortly after logging in.  I have to break the Mekanism pipe at the controller and then reconnect it. Breaking it earlier in the chain will not do it I must break and reconnect at the Refined Storage controller. The Pipes and Generators for power are all in the exact same chunk only 8 blocks away. 8 Block run of Mekanism Elite Universal Cable. Again my Refined Storage is not overly complicated as I am just getting started with it in this pack. 2 crafting grid, 1 pattern grid, 4 crafters with only one attached to a furnace to autocraft stuff for Refined storage, was 1 now 2 external storage bus with one in a different chunk 1 in the same chunk. 1 importer on the redstone furnace and 46 cables. I have not tried just simply hooking up external storage to a chest but can.

**Comment by Goldy93** [comment]
> 
> 
> have a look in the pack to see if there is other chunk loader options as this maybe related to the same bug that is in 1.12 with ftb chunk loader and RS
> 
> easy way to test is to disable the chunk loading log out and log back in and see if it happens
> 
> https://www.reddit.com/r/feedthebeast/comments/dkvuiu/refined_storage_not_detecting_my_storage_drawers/

I removed FTB chunks and Chicken Chunks, so no chunk loaders at all and no change, still experience the same issue when logging/relogging.  Going into the disk drive and switching to redstone only mode and then back again restores the connection to the drawer controller.  In my setup, the drawer controller is 2 chunks away.  I then moved it to the same chunk and no change, same issue.

**Comment by Demolishun** [comment]
I had this issue too.  I created a CC Tweaked computer.  At startup it runs a script that turns on a redstone signal to the External Storage interface for five seconds and then turns off.  The interface is configured to work with low redstone signal.  So sending a redstone signal disables the interface when server restarts.  I have not had this issue since I implemented this.  Also, I think I only experienced this issue with compacting drawers.  I "think" other drawers were not affected.   I am not entirely sure though.
